internal/auth/org/org.go

```diff
@@ -0,0 +1,218 @@
+package org
```

Comment:
Just a comment. I was wondering if `orgs` is better, but I myself has not been consistent on this..